core/json/src/main/scala/net/liftweb/json/JsonAST.scala

```diff
@@ -122,28 +208,44 @@ object JsonAST {
       case _ => false
     }
 
-    /** Return nth element from JSON.
-      * Meaningful only to JArray, JObject and JField. Returns JNothing for other types.
-      * <p>
-      * Example:<pre>
-      * JArray(JInt(1) :: JInt(2) :: Nil)(1) == JInt(2)
-      * </pre>
-      */
+    /**
+     * Return `n`th element from a `JArray` or `JObject` and return `JNothing` for any other kind of `JValue`.
```

Comment:
Wow. Looks like the original docs were wrong:

```
scala> val obj = JObject(JField("bacon", JInt(2)) :: JField("apple", JInt(3)) :: Nil)
obj: net.liftweb.json.JsonAST.JObject = JObject(List(JField(bacon,JInt(2)), JField(apple,JInt(3))))

scala> obj(1)
res1: net.liftweb.json.JsonAST.JValue = JNothing
```

So that's a fun discovery. Updated accordingly.